File: front/src/app/shared/models/Cocktail.ts
export class Cocktail {
    constructor(
        public id: number,
        public name: string,
        public alternativeName: string,
        public tags: string[],
        public category: string,
        public iba: string,
        public isAlcoholic: boolean,
        public glass: string,
        public instructions: string[],
        public thumbnail: string,
        public ingredients: string[],
        public measures: string[]
    ) { } 
}

File: back/Models/Cocktail.cs
namespace ShakeItUp.Models;

public class Cocktail
{
    public required int Id { get; set; }

    public required string Name { get; set; }

    public string? AlternateName { get; set; }

    public List<string> Tags { get; set; } = new();

    public required string Category { get; set; }

    public string? IBA { get; set; }

    public required bool Alcoholic { get; set; }

    public required string Glass { get; set; }

    public required string Instructions { get; set; }

    public required Uri Thumbnail { get; set; }

    public required List<string> Ingredients { get; set; }

    public required List<string> Measures { get; set; }
}
